src/Network/Network/ChangeLog.md

```diff
@@ -19,6 +19,12 @@
 --->
 
 ## Upcoming Release
+* Register App Service Resources ('Microsoft.Web/sites', 'Microsoft.Web/hostingEnvironments') for Private Link Powershells.
```

Comment:
The private link registration is a special case where changelog description is no need.